CoroutineTemplate/build.gradle

```diff
@@ -19,19 +19,20 @@ buildscript {
         androidx_navigation_version = '2.3.4'
         androidx_support_version = '1.3.0'
 
-        compose_version = '1.0.0-rc01'
+        compose_version = '1.0.2'
         constraint_layout_version = '2.0.0-alpha3'
 
-        hilt_version = '2.37'
+        hilt_version = '2.38.1'
 
-        kotlin_version = '1.5.10'
+        kotlin_version = '1.5.21'
         kotlinx_coroutines_version = '1.5.0'
 
         moshi_version = '1.12.0'
 
         okhttp_version = '4.9.1'
         retrofit_version = '2.9.0'
 
+        mockito_version = '3.12.4'
```

Comment:
I think we can remove `Mockito` for now because we didn't implement  the `Test` yet, Also  should we consider to use `Mockk` instead `Mockito` or What do you think? 🙏

Reply:
@Wadeewee It makes sense to remove the `Test` dependencies. May I know why we should consider to use `Mockk` instead of `Mockto` ?

Reply:
Mockito was evolved from Java; while Mockk is pure Kotlin; there are a few things you can do with Mockk i.e mocking the singleton 😄  we had a presentation somewhere I think @Wadeewee can point you to her/Toby's presentation back then.

Reply:
@doannimble You find the presentation [here](https://docs.google.com/presentation/d/1pSrDeMFRsSXq9gS58o7y3G5dnAKCUJlQn7EPWlaqQgo/edit) 🙏

Reply:
@Tuubz Tu
Thank you so much, I have a task to create basic unit test. I will use this for my reference 🥇